testsuite/replay-verify/archive_disk_utils.py

```diff
@@ -506,7 +506,7 @@ def create_one_pvc_from_snapshot(
 ) -> str:
     config.load_kube_config()
     api_instance = client.CoreV1Api()
-    storage_size = "10Ti" if TESTNET_SNAPSHOT_NAME in snapshot_name else "8Ti"
+    storage_size = "10Ti" if TESTNET_SNAPSHOT_NAME in snapshot_name else "10Ti"
```

Comment:
You don't need if else here?

Reply:
leave this for future changes on disk size